plugin/src/main/java/com/denizenscript/denizen/objects/properties/material/MaterialDisarmed.java

```diff
@@ -0,0 +1,103 @@
+package com.denizenscript.denizen.objects.properties.material;
+
+import com.denizenscript.denizen.objects.MaterialTag;
+import com.denizenscript.denizencore.objects.Mechanism;
+import com.denizenscript.denizencore.objects.ObjectTag;
+import com.denizenscript.denizencore.objects.core.ElementTag;
+import com.denizenscript.denizencore.objects.properties.Property;
+import com.denizenscript.denizencore.objects.properties.PropertyParser;
+import org.bukkit.Bukkit;
+import org.bukkit.block.data.BlockData;
+import org.bukkit.block.data.type.Tripwire;
+
+public class MaterialDisarmed implements Property {
+
+    public static boolean describes(ObjectTag material) {
+        if (!(material instanceof MaterialTag)) {
+            return false;
+        }
+        MaterialTag mat = (MaterialTag) material;

```

Comment:
Can use modern java `instanceof` format (I.e. `instanceof MaterialTag mat`) instead of manually casting to create the variable